tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -346,6 +346,243 @@ def test_end_date_gte_lte(self, url, expected_dag_run_ids, session):
         self.assertEqual(dag_run_ids, expected_dag_run_ids)
 
 
+class TestGetDagRunBatch(TestDagRunEndpoint):
+    @provide_session
+    def test_should_respond_200(self, session):
+        dag_runs = self._create_test_dag_run()
+        session.add_all(dag_runs)
+        session.commit()
+        payload = {
+            "dag_ids": ["TEST_DAG_ID"],
+        }
+        response = self.client.post("api/v1/dags/~/dagRuns/list", json=payload)
+        assert response.status_code == 200
+        self.assertEqual(
```

Comment:
Should we use the same style of asserts in test? I would opt for `assert x == y`

Reply:
Yes. Replaced `assertEqual` with `assert`